rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/admin/fossology/FossologyAdminController.java

```diff
@@ -126,4 +132,43 @@ public ResponseEntity<?> checkServerConnection() throws SW360Exception {
         return ResponseEntity.ok(Series.SUCCESSFUL);
 
     }
+
+    @Operation(
+            summary = "FOSSology connection configuration data.",
+            description = "Get the FOSSology connection configuration data.",
+            tags = {"Admin"},
+            responses = {
+            @ApiResponse(
+                    responseCode = "200", description = "Connection Configuration data",
+                    content = {
+                            @Content(mediaType = "application/json",
+                                    schema = @Schema(
+                                            example = """
+                                                    {
+                                                       "url": [
+                                                         "http://localhost:8000/"
+                                                       ],
+                                                       "folderId": [
+                                                         "1"
+                                                       ],
+                                                       "token": [
+                                                         "Wk0eiIsInNjb3BlIjoid3JpdGUifQ"
+                                                       ]
```

Comment:
Sending the token is a security risk. I know the current UI displays it, but I am not very comfortable about it.

Reply:
Either we can encrypt it before saving it in db or we can encrypt the data(token field) for this endpoint.

Reply:
I would say lets not send the token at all in the response. Clients does not need to know it, all they need to know if it is set or not.

Reply:
Understood. I hope this is ok, 
![image](https://github.com/user-attachments/assets/61c4bfb3-39bb-486e-88e9-c8505604a150)

Reply:
Would be great 👌

Reply:
But why mixing camel and snake case? I mean `folderId` and `token_set` in same response. Please be consistent

Reply:
My bad, made the changes.